src/main/java/org/jitsi/videobridge/shim/ConferenceShim.java

```diff
@@ -216,10 +216,16 @@ public void processOctoChannels(
         {
             tentacle.expire();
         }
-        else
+        else if (audioChannel instanceof ColibriConferenceIQ.OctoChannel
```

Comment:
should this re-use `VideobridgeShim#isOctoChannel`?

Reply:
No, this is intentional. The methods to get the relays are only available in the OctoChannel class.